operations/helm/charts/mimir-distributed/templates/ingester/ingester-statefulset.yaml

```diff
@@ -15,6 +15,15 @@ metadata:
 spec:
   podManagementPolicy: {{ .Values.ingester.podManagementPolicy }}
   replicas: {{ $rolloutZone.replicas }}
+  {{- if and (semverCompare ">= 1.23-0" .KubeVersion) (.Values.ingester.persistentVolume.enableStatefulSetAutoDeletePVC)  }}
```

Comment:
I don't think that `.KubeVersion` can be used like this. Here's an example usage https://github.com/grafana/mimir/blob/a6caa03b4a05935140c53eef974bbadcf02787c3/operations/helm/charts/mimir-distributed/templates/_helpers.tpl#L381